guides/languages.md

```diff
@@ -32,10 +32,9 @@ branch.
 - SPSS (legacy)
  
 ## Platform Management
-- Powershell (for DevOps / orchestration)
-- CLI
+- Powershell
+- Bash
 - ARM
-- Bat files
```

Comment:
As above, if still in use, let's leave it in.

Reply:
I think my main point that it wasn't a language.

Reply:
I'm pretty sure Batch is a language (known as "MS DOS Batch Script"). It's just a really minimal one - adding maybe a dozen commands to the DOS shell to support control flow.